src/test/java/com/epam/tc/hw3/testcases/Exercise1.java

```diff
@@ -0,0 +1,128 @@
+package com.epam.tc.hw3.testcases;
+
+import com.epam.tc.hw3.library.pages.Header;
+import com.epam.tc.hw3.library.pages.HomePage;
+import com.epam.tc.hw3.library.pages.SideBar;
+import com.epam.tc.hw3.library.utils.GetProperties;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.PageFactory;
+import org.testng.annotations.Test;
+
+public class Exercise1 extends BaseTest {
+
+    //1. Open test site by URL - in parent class
+    @Test
+    public void exercise1() {
+        //2. Assert Browser title
+        SoftAssertions softly = new SoftAssertions();
+        softly.assertThat(testDriver.getTitle()).as("Incorrect browser title").isEqualTo("Home Page");
+
+        Header header = PageFactory.initElements(testDriver, Header.class);
+        //3. Perform login
```

Comment:
Please, make this code according to Page Element(Component) pattern. Lecture 3, since 1:00:00. You should modify your page classes. Also, this info contains in https://habr.com/ru/company/jugru/blog/338836/ article

Reply:
Fixed